test/commonjs/basic/rewrite-route-prefix/two/index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = async (server, opts) => {
-  server.get('/', async (req, reply) => {
+  server.get('/two', async (req, reply) => {
```

Comment:
Funnily we have fixed an underlining bug in Fastify v4 - this change is actually correct, without it now throws an error.